Source/carthage/Outdated.swift

```diff
@@ -54,13 +55,40 @@ public struct OutdatedCommand: CommandProtocol {
 
 				if !outdatedDependencies.isEmpty {
 					carthage.println(formatting.path("The following dependencies are outdated:"))
+
 					for (project, current, updated, latest) in outdatedDependencies {
+						let versionColor: Color.Named.Color
+						switch (current, updated, latest) {
+						case (_, updated, latest) where updated == latest:
+							// Update available and applicable
+							versionColor = .green
+						case (current, updated, latest) where current != updated && updated != latest:
+							// Update availabe and applicable, but not to the latest version due to version lock
+							versionColor = .yellow
+						case (current, updated, latest) where current == updated:
+							// Update available, but not applicable due to version lock
+							versionColor = .red
+						default:
+							versionColor = .white
+						}
+
 						if options.outputXcodeWarnings {
-							carthage.println("warning: \(formatting.projectName(project.name)) is out of date (\(current) -> \(updated)) (Latest: \(latest))")
+							carthage.println("warning: \(formatting.projectName(project.name)) is out of date (\(current) -> \(updated) (Latest: \(latest))_")
 						} else {
-							carthage.println(formatting.projectName(project.name) + " \(current) -> \(updated) (Latest: \(latest))")
+							let versionSummary = formatting.colored(current.description, color: versionColor)
+								+ " -> " + formatting.colored(updated.description, color: versionColor)
+								+ " (Latest: \(latest))"
+							carthage.println(formatting.projectName(project.name) + " " + versionSummary)
 						}
 					}
+
+					if options.colorOptions.formatting.isColorful {
+						carthage.println(formatting.path("The color indicates what happens when you run `carthage update`"))
+						carthage.println(formatting.colored("<green>", color: .green) + "\t\t- Will be updated to the newest version")
+						carthage.println(formatting.colored("<yellow>", color: .yellow) + "\t- Will be updated, but not to the newest version"
+							+ "because of the specified version in Cartfile")
```

Comment:
Need a space before `because` to separate it from `version` (see your screenshot of the example output)

Reply:
Oh yeah, right 👍